packages/deployments/contracts/contracts/messaging/connectors/polygon/lib/Merkle.sol

```diff
@@ -18,7 +18,7 @@ library Merkle {
     bytes32 proofElement;
     bytes32 computedHash = leaf;
     uint256 len = proof.length;
-    for (uint256 i = 32; i <= len; ) {
+    for (uint256 i = 32; i <= len + 1; ) {
```

Comment:
I think this should be 
`for (uint256 i = 32; i < len + 1; ) { `
or
`for (uint256 i = 32; i <= len; ) {`

Reply:
fixed!